### PR TITLE
Minor hardware detail tweaks

### PR DESCRIPTION
**Pull request by dhellmann**

### CODE REVIEW

**Comment by dhellmann** [comment]
This will need a rebase and we should merge #11 first.

**Comment by russellb** [comment]
Merged #11, so now this one needs a rebase

**Comment by dhellmann** [comment]
Rebased

File: pkg/apis/metalkube/v1alpha1/baremetalhost_types.go [hunk 1]
```diff
@@ -71,25 +71,27 @@ type BareMetalHostSpec struct {
...
-	Speed int    `json:"speed"` // GHz
+	Type     string `json:"type"`
+	SpeedGHz int    `json:"speedGHz"`
 }
 
 // Storage describes one storage device (disk, SSD, etc.) on the host.
 type Storage struct {
-	Size int    `json:"size"` // GB
-	Info string `json:"info"` // model, etc.
+	SizeGIB int    `json:"sizeGIB"`
```
Comment (russellb) [inline pkg/apis/metalkube/v1alpha1/baremetalhost_types.go]: GIB here, by GB for NIC speed and RAM amount?  Intentional?

Comment (dhellmann) [reply pkg/apis/metalkube/v1alpha1/baremetalhost_types.go]: Updated

**Comment by russellb** [comment]
From a discussion with @andybraren

What a GiB is: https://searchstorage.techtarget.com/definition/gibibyte-GiB
Ubuntu’s Units Policy: https://wiki.ubuntu.com/UnitsPolicy
Windows vs Ubuntu reporting: https://www.hecticgeek.com/2012/10/for-the-same-file-why-windows-and-ubuntu-show-different-sizes/
Kubernetes supports both units: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory
Cockpit discussion: https://github.com/cockpit-project/cockpit/issues/704
Cockpit conclusion: https://github.com/cockpit-project/cockpit/pull/3944#issuecomment-194724919

**Pull request merged**